downgrade

```diff
@@ -176,6 +187,7 @@ main() {
 
   for term; do
     installed=( $(previously_installed "$term") )
+    current=( $(currently_installed "$term") )
```

Comment:
This makes current a single-element array (well it files it, you've `declare`d it as an array below). But then you treat it like a string with `-z` and `*$current*` above. I think you just want to use

```sh
current=$(currently_installed "$term")
```

Here, and drop the `declare` below.